test/Window/WindowBase.test.cpp

```diff
@@ -48,6 +48,25 @@ TEST_CASE("[Window] sf::WindowBase", runDisplayTests())
             CHECK(windowBase.getSize() == sf::Vector2u(360, 240));
             CHECK(windowBase.getSystemHandle() != sf::WindowHandle());
         }
+
+        SECTION("Move construction")
+        {
+            sf::WindowBase       movedWWindowBase(sf::VideoMode({360, 240}), "WindowBase Tests");
+            const sf::WindowBase windowBase(std::move(movedWWindowBase));
+            CHECK(windowBase.isOpen());
+            CHECK(windowBase.getSize() == sf::Vector2u(360, 240));
+            CHECK(windowBase.getSystemHandle() != sf::WindowHandle());
```

Comment:
Heads up, we renamed `getSystemHandle` to `getNativeHandle`.

Reply:
Thanks, I will fix it. and learn the rebase as you mentioned. 👍